pages/faq.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: default
+title: FAQ
+permalink: /FAQ/
+nav_page: whatWeDo
+nav_page_specific: faq
+---
+
+# Frequently Asked Questions
+
+<hr class="bg-primary"/>
+
+Have a question about AB Tech, getting involved, or hiring us? We have answers (well, most of the time). Don't see your question below? Please email <a href = "mailto: abtech@andrew.cmu.edu">abtech@andrew.cmu.edu</a> with any questions you may have.
+
+- <a href="#General">General</a>
+- <a href="#Getting Involved">Getting Involved</a>
+- <a href="#Requesting AB Tech">Requesting AB Tech</a>
+
+<h2>
```

Comment:
Change this `h2` block to just be `## General`. You should be able to link to it as `#general`.

Reply:
`[## General](#General)` like so?

Reply:
`[## General](#general)`

Reply:
<img width="445" alt="image" src="https://user-images.githubusercontent.com/96501599/151719897-8d03f9aa-96bb-4b98-abc9-870155a35060.png">
<img width="274" alt="image" src="https://user-images.githubusercontent.com/96501599/151719928-ddbea97e-a6e9-41b1-ab58-830a0383a4ef.png">


I can get markdown working for General and Getting Involved. Physically "requesting AB Tech" looks correct in the dev environment but doesn't scroll like others. On first glance does anything look incorrect about the way it's set up?

Reply:
The heading is created by doing:
```markdown
## My Heading
```
Jekyll Markdown turns this into HTML: `<h2 id="my-heading">My Heading</h2>`. This markdown (above) creates an anchor of the name `my-heading` which can be linked to like:
```markdown
[Link to jump to my heading](#my-heading)
```
The `[This is my link](#some-location)` Markdown syntax is the same thing as `<a href="#some-location">This is my link</a>` in HTML. Links to anchors begin with `#` and then the anchor name. You can also do other links like `[Google](https://google.com)` which turns into `<a href="https://google.com">Google</a>`.

So for the first image, these are links to jump to other parts of the page (I presume), so you would want:
```markdown
[General](#general)
[Getting Involved](#getting-involved)
[Requesting AB Tech](#requesting-ab-tech)
```
You don't need any HTML for the links. And then later where the heading actually goes you can do:
```markdown
## Requesting AB Tech

Blah blah this is how you request...
```

Reply:
The simple rules for how a heading is turned into an anchor are:
 - Spaces become dashes
 - Uppercase becomes lowercase